src/components/layout/TabBar.js

```diff
@@ -98,5 +98,5 @@ const Underline = styled.View`
   border-top-width: ${Platform.OS === 'android' ? '3px' : '6px'};
   border-top-left-radius: 3px;
   border-top-right-radius: 3px;
-  border-color: ${({ theme }) => theme.colors.primaryAccent130};
+  border-color: #A020F0;
```

Comment:
Could you check that these colour is not stored in the theme?

Reply:
yeah i checked and one is there on the dark theme but its not there on light theme. but we needed the same color irrespective on the dark mode or light mode right?

Reply:
It looks this way yes. If it exists for dark but not light, please add it for the light theme and use the theme reference instead.

Reply:
yeah okay

Reply:
Thanks 🙏